examples/plot_variogram_estimation_modelling.py

```diff
@@ -33,23 +25,19 @@
 import geoutils as gu
 
 # %%
-# We start by loading example files including a difference of DEMs at Longyearbyen and the outlines to rasterize
-# a glacier mask.
-# Prior to differencing, the DEMs were aligned using :ref:`coregistration_nuthkaab` as shown in
-# the :ref:`sphx_glr_auto_examples_plot_nuth_kaab.py` example. We later refer to those elevation differences as *dh*.
+# We load example files.
 
 dh = xdem.DEM(xdem.examples.get_path("longyearbyen_ddem"))
 glacier_outlines = gu.Vector(xdem.examples.get_path("longyearbyen_glacier_outlines"))
 mask_glacier = glacier_outlines.create_mask(dh)
 
 # %%
-# We remove values on glacier terrain in order to isolate stable terrain, our proxy for elevation measurement errors.
+# We exclude values on glacier terrain in order to isolate stable terrain, our proxy for elevation errors.
 dh.data[mask_glacier] = np.nan
 
 # %%
-# We estimate the average per-pixel elevation measurement error on stable terrain, using both the standard deviation
-# and normalized median absolute deviation. For this example, we do not account for the non-stationarity in elevation
-# measurement errors quantified in :ref:`sphx_glr_auto_examples_plot_nonstationary_error.py`.
+# We estimate the average per-pixel elevation error on stable terrain, using both the standard deviation
+# and normalized median absolute deviation. For this example, we do not account for elevation heteroscedasticity.
 print('STD: {:.2f} meters.'.format(np.nanstd(dh.data)))
 print('NMAD: {:.2f} meters.'.format(xdem.spatialstats.nmad(dh.data)))
```

Comment:
There's an annoying Warning that shows up in the documentation. Maybe disable warnings at the beginning?

Reply:
I guess the issue is in nmad. We could also try to fix that warning there.

Reply:
Solved with `set_mask` above, and then `np.ma.std`